dev/io.openliberty.jakarta.faces.4.0/src/META-INF/resources/jakarta.faces/faces-development.js

```diff
@@ -5309,7 +5313,7 @@ exports.Style = Style;
  * small helper for the specialized jsf case
  * @constructor
  */
-const DEFAULT_WHITELIST = () => {
+const DEFAULT_allowList = () => {
```

Comment:
allowlist should be uppercase here: DEFAULT_ALLOWLIST

Reply:
Thanks!